UsersApp/Extensions/UsersImageExtensions.swift

```diff
@@ -17,7 +17,7 @@ extension UIImageView {
                 let data = data, error == nil,
                 let image = UIImage(data: data)
             else {
-                let defaultImage = UIImage(systemName: "person")
+                let defaultImage = UIImage(systemName: "person.fill")
```

Comment:
I'am sure that you are familiar with SOLID principals, if not please read about it. 
Checking of downloaded image should be done in controller not in downloading method of the image. This method should return image if request has succeeded or an error if not.